guides/create-graphql-query.md

```diff
@@ -0,0 +1,239 @@
+## At a glance
+
+You're working on a new implementation for Leader of the Pack, an outdoor equipment retailer. Let's say that you don't ship Kayaks to every location because of the logistical challenges of shipping something that large. So you need to add a query that will show you if a particular product can be shipped to a particular zip code. Once you have that data in the system you need to have a way to make it available to the client. This is usually done by creating a GraphQL query. In this guide, we’ll walk you through the steps you need to create a query in GraphQL and have it recognized by the system.
+
+## What you need
+
+You should already have an understanding of what fields/data you wish to share with the client and what GraphQL type each field is. Consult the [GraphQL Reference](https://graphql.org/learn/schema/) for more info on GraphQL types.
+
+## Identify which plugin owns the query
+
+The complete Open Commerce GraphQL API is created by stitching together domain-specific APIs from all the API plugins. So when adding a new query, the first step is to decide which plugin should own it. This is usually obvious, but not always. You should think about whether any other plugins or services will need to call your query.
+Typically, if you are adding data your new query will exist in your custom plugin.
+
+## Name the query
+
+When choosing a name for the query, there are a few rules to follow:
+- In keeping with general GraphQL best practices, do not use verbs such as "list", "get", or "find" at the beginning of your query name. For example, use "cart" instead of "getCart" and "carts" instead of "listCarts".
+- Prefix with adjectives as necessary to fully describe what the query returns. For example, "anonymousCart" and "accountCart" queries.
+- If there are similar queries that take slightly different parameters, add a suffix to clarify. In most cases, we begin the suffix with "By". For example, "accountCartById" and "accountCartByAccountId".
+
+## Define the query in the schema
+
+- If it doesn't already exist, create `schemas` folder in the plugin, and add an `index.js` file there.
+- If it doesn't already exist, create `schema.graphql` in `schemas` in the plugin.
+- Import the GraphQL file into `index.js` and default export it in an array:
+
+```js
+import importAsString from "@reactioncommerce/api-utils/importAsString.js";
+
+const schema = importAsString("./schema.graphql");
+
+export default [schema];
+```
+
+> NOTE: For large plugins, you can split to multiple `.graphql` files and export a multi-item array.
+
+- In the `.graphql` file, add your query within `extend type Query { }`. Add an `extend type Query` section near the top if the file doesn't have it yet.
```

Comment:
Think we can remove most of the bullets from this section. Text/copy is perfectly fine, just doesn't feel like it should be living under bullets based on what's on /Developer at the moment